src/schema-gen/readme.md

```diff
@@ -2,45 +2,89 @@
 
-Depends on datatype definitions in subdirectory ../../../schema/. 
+Depends on datatype definitions in subdirectory (submodule) `../support/metaschema/schema`. 
 
-## xsl
+See the file manifest for a [survey of file contents](manifest.md) by file.
 
-### make-json-schema-metamap.xsl
 
-- XSLT stylesheet version 3.0 (63 templates)
-- **Purpose:** Produce an XPath-JSON document representing JSON Schema declarations from Metaschema source data. The results are conformant to the rules for the XPath 3.1 definition of an XML format capable of being cast (using the xml-to-json() function) into JSON.
-- **Note:** this XSLT will only be used on its own for development and debugging. It is however imported by `produce-json-converter.xsl` and possibly other stylesheets.
+Currently, schemas in two formats are supported, with a third under development:
 
-### make-metaschema-metatron.xsl
+- XSD (version 1.1)
+- JSON Schema (v 7)
+- Schematron - Metaschema Constraints model (in development) - code-name *Metatron*
 
-- XSLT stylesheet version 3.0 (34 templates)
-- **Purpose:** Produce an Schematron representing constraints declared in a metaschema
-- **Input:** A Metaschema
-- **Output:** An XSD, with embedded documentation
-- Runtime parameter `$produce-warnings` as xs:string
-- Runtime parameter `$noisy` as xs:string
-- Runtime parameter `$debug` with default ''no''
+## Producing schemas for a single metaschema source
 
-- Compile-time dependency (xsl:import) `../metapath/parse-metapath.xsl`
-- Compile-time dependency (xsl:import) `metatron-datatype-functions.xsl`
+Schema generation from NIST Metaschema source data is currently supported by XSLT in this subdirectory. This can be done in any of several ways as is convenient for a developer, tester or Metaschema end user (metaschema author or developer):
 
-### make-metaschema-xsd.xsl
+- **make** files supporting the testing and production infrastructure
+- **Command line scripts** calling Apache Maven for Java configuration and support
+- **calling XSLT directly** the same transformations can be executed directly from an environment already supporting XSLT
+- **using XProc** the same transformations are orchestrated using XProc for efficiency and speed
 
-- XSLT stylesheet version 3.0 (41 templates)
-- **Purpose:** Produce an XSD Schema representing constraints declared in a metaschema
-- **Input:** A Metaschema
-- **Output:** An XSD, with embedded documentation
-- Runtime parameter `$debug` with default ''no''
+Currently, all these assume a single top-level metaschema as input, with its modules called via `import`.
 
-### metatron-datatype-functions.xsl
+Routines that provide the same services over sets or batches of metaschemas, for efficiency, are to be developed.
+### Produce an XSD using `make`
 
-- XSLT stylesheet version 3.0 (7 templates)
-- Can be run standalone, but also serves as a utility XSLT to `make-metaschema-metatron.xsl`
+XXX tbd XXX
```

Comment:
Missing information. I understand @Nikita offered towork on it, but not in the next 2 weeks, I assume.

Reply:
We are by no means done with all this - here we are starting, not stopping, and there are indeed follow up Issues where we will revisit scripting and docs, including #41 and #42 ...